google-cloud-bigtable/lib/google/cloud/bigtable/admin/v2/bigtable_instance_admin_client.rb

```diff
@@ -1000,15 +992,15 @@ def list_clusters \
             #   operation.wait_until_done!
 
             def update_cluster \
-                name,
                 serve_nodes,
+                name: nil,
```

Comment:
This breaking change will require an update in the manual layer.

Reply:
@quartzmo I think this breaking change is incorrect. Let me inquire upstream.